draft-ietf-quic-multipath.md

```diff
@@ -710,32 +706,28 @@ This also means that the same packet number can occur on each path and the
 packet number is not a unique identifier anymore. This requires changes to
 the ACK frame as well as packet protection as described in the following subsections.
 
-When multipath is negotiated,
-each path ID is linked to a separate packet number space.
+When multipath is negotiated, separate packet number space is linked to a Path ID.
 Each Path ID-specific packet number space starts at packet number 0. When following
 the packet number encoding algorithm described in {{Section A.2 of QUIC-TRANSPORT}},
 the largest packet number (largest_acked) that has been acknowledged by the
-peer in this new connection ID's packet number space is initially set to "None".
+peer in the Path ID-specfic packet number space is initially set to "None".
 
 ## Sending Acknowledgements
 
 The ACK_MP frame, as specified in {{ack-mp-frame}}, is used to
 acknowledge 1-RTT packets.
-Compared to the QUIC version 1 ACK frame, the ACK_MP frame additionally
-contains the receiver's Path ID associated with the Destination Connection ID
-to distinguish the path-specific packet number space.
-
-Acknowledgements of Initial and Handshake packets MUST be carried using
-ACK frames, as specified in {{QUIC-TRANSPORT}}. The ACK frames, as defined
-in {{QUIC-TRANSPORT}}, do not have a
-Path Identifier field to identify the path and this the packet number space.
```

Comment:
MUST has been removed because this is implicit as ACK_MP frames cannot be used before negotiation succeeded. Also the second sentence was removed because it is redundant.